# Unified diff: 4a9a4f0:vnn_config/<bench>.ini  vs  benchmarks/vnn_files/vnn_config_2025_competition/<bench>.ini
# (only the trivial parser-compat renames; semantically identical to competition)

--- 4a9a4f0:vnn_config/cersyve.ini
+++ vnn_config_2025_competition/cersyve.ini
8c8
< step_ratio=0.1
---
> lr_attack=0.1
10c10
< nb_repeat=10000
---
> nb_restart=10000

--- 4a9a4f0:vnn_config/cifar100_2024.ini
+++ vnn_config_2025_competition/cifar100_2024.ini
10c10
< nb_repeat=10
\ No newline at end of file
---
> nb_restart=10
\ No newline at end of file

--- 4a9a4f0:vnn_config/cora_2024.ini
+++ vnn_config_2025_competition/cora_2024.ini
10c10
< nb_repeat=100
\ No newline at end of file
---
> nb_restart=100
\ No newline at end of file

--- 4a9a4f0:vnn_config/relusplitter.ini
+++ vnn_config_2025_competition/relusplitter.ini
10c10
< nb_repeat=10
\ No newline at end of file
---
> nb_restart=10
\ No newline at end of file

--- 4a9a4f0:vnn_config/safenlp_2024.ini
+++ vnn_config_2025_competition/safenlp_2024.ini
10c10
< nb_repeat=100
---
> nb_restart=100

--- 4a9a4f0:vnn_config/sat_relu.ini
+++ vnn_config_2025_competition/sat_relu.ini
1c1
< nb_repeat=10000
---
> nb_restart=10000

--- 4a9a4f0:vnn_config/soundnessbench.ini
+++ vnn_config_2025_competition/soundnessbench.ini
9c9
< nb_repeat=10
\ No newline at end of file
---
> nb_restart=10
\ No newline at end of file

--- 4a9a4f0:vnn_config/tinyimagenet_2024.ini
+++ vnn_config_2025_competition/tinyimagenet_2024.ini
10c10
< nb_repeat=10
\ No newline at end of file
---
> nb_restart=10
\ No newline at end of file

--- 4a9a4f0:vnn_config/tllverifybench_2023.ini
+++ vnn_config_2025_competition/tllverifybench_2023.ini
7c7
< step_ratio=0.1
---
> lr_attack=0.1

--- 4a9a4f0:vnn_config/traffic_signs_recognition_2023.ini
+++ vnn_config_2025_competition/traffic_signs_recognition_2023.ini
5c5
< nb_repeat=2000
---
> nb_restart=2000

